web/character/investigation.py

```diff
@@ -1381,7 +1384,7 @@ def clue_discoveries(self):
     def func(self):
         """Executes clues command"""
         try:
-            if not self.args or "search" in self.switches:
+            if not self.args or "search" in self.switches or "search/tag" in self.switches or "search/name" in self.switches:

```

Comment:
I don't think this will work - I believe that switches split based on the "/" separator, so for those it would be that "search" is in self.switches and "tag" is in self.switches. I'll need to double check that

Reply:
Belatedly, but that seems to be correct (needing to check for "B in self.switches and C in self.switches" after confirming A).  Example of this is in the flashback code with "flashback/check/flub" effectively doing that.  (It checks for ("check","roll") first, then calls a function which checks for "flub" on its own.  See scene_commands.py lines 69 and 255.)

Because of that, I don't think it's necessary to even look for /tag or /name, since they're subordinate to /search.

```suggestion
            if not self.args or "search" in self.switches:
```